docs/parameters.rst

```diff
@@ -49,5 +49,16 @@ Here is the full list of configuration parameters you can specify in a ``config.
  	``'segmentation'``
  		Output is an array of shape ``(256, 256)`` with values matching the class index label at that position. The classes are applied sequentially according to ``config.json`` so latter classes will be written over earlier class labels if there is overlap.
 
+**seed**: int
+    Random generator seed. Optional, use to make results reproducible.
+
+**split_vals**: list
+    Default: `[0.8, 0.2]`
+    Percentage of data to put in each catagory listed in split_names. Must be floats and must sum to one.
```

Comment:
```suggestion
    Percentage of data to put in each category listed in split_names. Must be a list of floats that sum to one and match the length of `split-names`. For train, validate, and test data, a list like `[0.7, 0.2, 0.1]` is suggested.
```

I think we can be a little more explicit in suggesting people use all types of data.